src/editor.jsx

```diff
@@ -369,7 +409,7 @@ var Editor = React.createClass({
                     var duplicate = id in widgets;
 
                     widgets[id] = this.getWidgetEditor(id, type);
-                    var classes = (duplicate || !widgets[id] ? "error " : "") +
+                    classes = (duplicate || !widgets[id] ? "error " : "") +
                             (selected ? "selected " : "");
```

Comment:
I removed "var" and declared "classes" at the start of the function.  This could probably be rewritten use ```cx``` to make this more React-y.